### PR TITLE
Eg gui brain atlas bugs

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

**Comment by egolol** [comment]
The java utility to make dynamic columns is not build into Matlab

File: braph2/atlas/BrainAtlas.m [hunk 1]
```diff
@@ -405,8 +406,8 @@ function save_to_xls(atlas, varargin)
             
             % creates table
             tab = [
-                {atlas.getID(), {}, {}, {}, {}, {}};
-                table(br_ids, br_label, br_notes, br_x, br_y, br_z)
+                {atlas.getID(), atlas.getBrainSurfFile(), {}, {}, {}, {}};
```
Comment (giovannivolpe) [inline braph2/atlas/BrainAtlas.m:409]: You should also save atlas.getLabel and atlas.getNotes()

File: braph2/atlas/BrainAtlas.m [hunk 2]
```diff
@@ -497,8 +499,8 @@ function save_to_txt(atlas, varargin)
...
```
Comment (giovannivolpe) [inline braph2/atlas/BrainAtlas.m:502]: Same here, save also atlas label and notes

File: braph2/atlas/GUIBrainAtlas.m [hunk 3]
```diff
@@ -237,6 +237,18 @@ function init_filename()
...
+        else
+            init_disable()
+        end
+    end
+
+    function init_disable()
+%         set(ui_checkbox_figure_distance, 'enable', 'off')
+    end
+    function init_enable()
+%         set(ui_checkbox_figure_distance, 'enable', 'on')
```
Comment (giovannivolpe) [inline braph2/atlas/GUIBrainAtlas.m:251]: remove

File: braph2/atlas/GUIBrainAtlas.m [hunk 4]
```diff
@@ -446,12 +458,16 @@ function cb_table_add(~, ~)  % (src, event)
         br = BrainRegion([num2str(br_id)], '', '', 0, 0, 0);
         atlas.getBrainRegions().add(br.getID(), br);
         update_table_table()
+        init_enable()
```
Comment (giovannivolpe) [inline braph2/atlas/GUIBrainAtlas.m:461]: remove

File: braph2/atlas/GUIBrainAtlas.m [hunk 5]
```diff
@@ -446,12 +458,16 @@ function cb_table_add(~, ~)  % (src, event)
...
         create_figure()
     end
     function cb_table_remove(~, ~)  % (src, event)
         selected = atlas.getBrainRegions().remove_all(selected);
         update_table_table()
         create_figure()
+        if atlas.getBrainRegions().length() == 0        
+            init_disable()
+        end
```
Comment (giovannivolpe) [inline braph2/atlas/GUIBrainAtlas.m:470]: remove

File: braph2/atlas/GUIBrainAtlas.m [hunk 6]
```diff
@@ -479,7 +495,7 @@ function cb_table_move2bottom(~, ~)  % (src, event)
 FIG_BRAIN_TP = 'Brain surface on/off';
 
 FIG_DISTANCE_CMD = 'Highlights';
-FIG_DISTANCE_TP = 'Highlights on/off';
+FIG_DISTANCE_TP = 'Highlights on/off. Must select a Brain Region first.';
```
Comment (giovannivolpe) [inline braph2/atlas/GUIBrainAtlas.m:498]: 'Highlights of selected brain regions on/off.'

File: braph2/atlas/GUIBrainAtlas.m [hunk 7]
```diff
@@ -1211,6 +1227,7 @@ function setup()
         update_table_table()
         % setup figure
         create_figure()
+        init_disable()
```
Comment (giovannivolpe) [inline braph2/atlas/GUIBrainAtlas.m:1230]: remove

**Pull request merged**